### PR TITLE
update to use icr/cpopen registry

### PR DESCRIPTION
**Pull request by siddarthpatel**
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/53255

### CODE REVIEW

File: bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -666,22 +666,22 @@ spec:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: COMMON_WEB_UI_IMAGE
-                  value: quay.io/opencloudio/common-web-ui:1.16.0
-                image: quay.io/opencloudio/ibm-commonui-operator:1.16.0
+                  value: icr.io/cpopen/common-web-ui:1.16.0
```
Comment (ashank07) [inline bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml:669]: This should be icr.io/cpopen/cpfs

File: bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -873,8 +873,8 @@ spec:
   provider:
     name: IBM
   relatedImages:
-  - image: quay.io/opencloudio/ibm-commonui-operator:1.16.0
+  - image: icr.io/cpopen/ibm-commonui-operator:1.16.0
     name: COMMON_WEB_UI_OPERATOR_IMAGE
-  - image: quay.io/opencloudio/common-web-ui:1.16.0
+  - image: icr.io/cpopen/common-web-ui:1.16.0
```
Comment (ashank07) [inline bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml:878]: This should be icr.io/cpopen/cpfs

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -58,19 +58,19 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: COMMON_WEB_UI_IMAGE
-              value: "quay.io/opencloudio/common-web-ui:1.16.0"
+              value: "icr.io/cpopen/common-web-ui:1.16.0"
```
Comment (ashank07) [inline config/manager/manager.yaml:61]: Should be icr.io/cpopen/cpfs

File: bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (sgrube) [reply bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml]: fixed

File: bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (sgrube) [reply bundle/manifests/ibm-commonui-operator.clusterserviceversion.yaml]: fixed

File: config/manager/manager.yaml (hunk 3 above)
Comment (sgrube) [reply config/manager/manager.yaml]: fixed

**Pull request merged**